README.md

```diff
@@ -54,7 +62,7 @@ Contributions, issues, and feature requests are welcome!
 
 Give a ⭐ if you like this project and how it is built!
 
-## 📝 License
 
-This project is [MIT](https://github.com/Smart1-hub/Smart Development/blob/add-license-1/LICENSE)
+## 📝 License
 
+This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] Let me explain what MIT is all about this is a license so this project is copyrighted for you, The way we are supposed to set it up is to download the template from [This link](https://github.com/microverseinc/readme-template/blob/master/MIT.md) in root directory of your project then remove boilerplate with suitable one and then link it in the READme file, Kindly implement that I didn't wish to block your progress over something that simple. Wish you good luck 😇